### PR TITLE
Feature/15 add create workspace dialog

### PR DESCRIPTION
**Pull request by zsinnema**
see #15

### CODE REVIEW

File: applications/osb-portal/src/apiclient/workspaces/models/InlineResponse2004.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Workspaces manager API
+ * MetaCell Cloudharness Platform - Reference Workspaces manager API
+ *
+ * The version of the OpenAPI document: 0.1.0
+ * Contact: mnp@metacell.us
```
Comment (filippomc) [inline applications/osb-portal/src/apiclient/workspaces/models/InlineResponse2004.ts:8]: Probably need to fix directly on the openapi file

File: applications/osb-portal/src/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,864 @@
+openapi: 3.0.0
+info:
+  contact:
+    email: mnp@metacell.us
```
Comment (filippomc) [inline applications/osb-portal/src/openapi.yaml:4]: @zsinnema change to opensourcebrain, or remove

File: applications/osb-portal/src/openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,864 @@
...
+  description: MetaCell Cloudharness Platform - Reference Workspaces manager API
```
Comment (filippomc) [inline applications/osb-portal/src/openapi.yaml:5]: OSB not cloudharness

**Comment by zsinnema** [comment]
done

**Pull request merged**